coffeecard/CoffeeCard.Models/DataTransferObjects/v2/Product/UpdateMenuItemRequest.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using CoffeeCard.Models.Entities;
+
+namespace CoffeeCard.Models.DataTransferObjects.v2.Product
+{
+    /// /// <summary>
+    /// Initiate an update product request.
+    /// </summary>
+    /// <example>
+    /// {
+    ///   "Id": 1,
+    ///   "Name": "Espresso",
+    /// }
+    /// </example>
+    public class UpdateMenuItemRequest
+    {
+        /// <summary>
+        /// Gets or sets the ID of the product to update.
+        /// </summary>
+        /// <value> Product Id </value>
+        /// <example> 1 </example>
+        [Required]
+        public int Id { get; set; }
```

Comment:
If we change to an API design like `PUT menuitems/[id]`, this property is not required